vnext/Microsoft.ReactNative.Cxx/UI.Xaml.impl.override.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+// In this file we override xaml::Application::Current() to return nullptr
+// instead of crashing when Application is not found.
+#pragma once
+#include <winrt/base.h>
+
+#ifndef USE_WINUI3
```

Comment:
I don't like this very much, could you elaborate on why this file is needed? 
Is this a fork of a cppwinrt-generated header? if so that seems fragile as we're going to update the version of WinUI3 every once in a while, and we shouldn't have to have to keep updating this file...

Reply:
The goal was to avoid to crash on the Application::Current() call. In this case we can do it differently. I will change it.

---
In reply to: [432993083](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432993083) [](ancestors = 432993083)

Reply:
Use TryGetCurrentApplication instead.

---
In reply to: [433250812](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433250812) [](ancestors = 433250812,432993083)